src/clj/game/core/ice.clj

```diff
@@ -320,6 +320,7 @@
 (defn get-strength
   [card]
   (when (or (ice? card)
+            (:strength card)
```

Comment:
how does this apply?

Reply:
It means we can do forced encounters with non-ice cards, so long as they have a listed strength.
There's an upgrade that pretends to be an ice on access, and it's surprising that this is the *only* change needed to make it work.

Reply:
in that case, can you make it more specific? either by name or by "upgrade and :strength"?

Reply:
I added a comment which explains the usage. We can get the (printed) strength of a card if it's ice, a program, or has a printed strength.